### PR TITLE
Feature/convert to monolith

### PR DESCRIPTION
**Pull request by codespool**
Having a monorepo setup didn't really show much benefit, and likely contributed to the @polkadot/* dependency confusion and issues.
I believe reverting back to the monolith setup and exposing the utility functions that are likely to be used by other projects will lead to simpler and more maintainable project. (and also simplify the switch to ESM.

NOTE FOR REVIEW:
- no need to check all the individual files, not much code has been changed, only directory structure and configs/paths

### CODE REVIEW

**Comment by codespool** [comment]
Update: 
So far, it builds and works as before. Only issue is with `./bin/dev` not working properly, probably related: https://github.com/oclif/oclif/issues/1016

File: README.md
Comment (shunsukew) [inline README.md:1]: Repository structure has been changed, README updates also needed in my opinion.

Comment (codespool) [reply README.md:1]: I'll update readme (and docs) once this, ESM and the patches are merged, so I don't have to rewrite it 3 times

**Review by shunsukew — APPROVED** [review]

**Comment by shunsukew** [comment]
Any impacts on ecosystem toolings using core packages?

**Comment by codespool** [comment]
> Any impacts on ecosystem toolings using core packages?

The package name will be different (@astar-network/swanky-cli), but the exports should stay the same.
Possibly needs a bit more config with the ESM PR merged.

**Pull request merged**